chandra_aca/dark_model.py

```diff
@@ -100,22 +96,24 @@ def get_dark_hist(date, t_ccd):
     """
     pars = get_sbp_pars(date)
     x = darkbins.bin_centers
-    y = smooth_broken_pow(pars, x)
+    y = smooth_twice_broken_pow(pars, x)
 
-    scale = dark_temp_scale(-19, t_ccd)
+    # Model params are calibrated using reference temp. -14 C
+    scale = dark_temp_scale(-14, t_ccd)
     xbins = darkbins.bins * scale
     x = x * scale
     return x, xbins, y
 
 
 def smooth_broken_pow(pars, x):
```

Comment:
I think this routine could also be cut.  Will it give the wrong answer at this point anyway?